src/watchdog.js

```diff
@@ -98,253 +96,115 @@ export default class Watchdog {
 			}
 		};
 
-		/**
-		 * Throttled save method. The `save()` method is called the specified `saveInterval` after `throttledSave()` is called,
-		 * unless a new action happens in the meantime.
-		 *
-		 * @private
-		 * @type {Function}
-		 */
-		this._throttledSave = throttle(
-			this._save.bind( this ),
-			typeof config.saveInterval === 'number' ? config.saveInterval : 5000
-		);
-
-		/**
-		 * The current editor instance.
-		 *
-		 * @private
-		 * @type {module:core/editor/editor~Editor}
-		 */
-		this._editor = null;
+		if ( !this._restart ) {
```

Comment:
It will crash much later - once the editor / context will crash. And the user integrating the watchdog feature may not notice it, hence I decided to throw earlier